### DISCUSSION TITLE
Conversion to pp format

### DISCUSSION THREAD

**Discussion by StephanieWoodward** [MetOffice/simulation-systems#170] [UM]
I want to look at some fields from from a UM suite which crashed after a few timesteps (the fields were written each timestep) but the file on the HPC has not yet been converted to pp format.   Could someone please remind me how to do this conversion?  It's a long time since I last did this and my current attempts are all failing, so an idiot-proof description would be very welcome :)

**Comment by richardgilham** [comment]
You'll can use mule-convpp for this. On the HPC...

`module load um_tools`

then

`mule-convpp --help`

Browser-based documentation can be found towards the bottom of this page: https://www-nwp/umdoc/ (Met Office internal only)

**Reply by StephanieWoodward** [reply]
Many thanks Richard - it's all working now.  (My mistake had been trying to run this on the Linux system )
